### DISCUSSION TITLE
Keyboard shortcuts to control zoom/chart centre

### DISCUSSION THREAD

**Discussion by boatybits** [antipole2/JavaScript_pi#85] [Ideas]
following on from the thread on cruisersforum, what I would find useful would be keyboard shortcuts to set chart scale and chart position. For instance, press a key and chart zoom is set to 1:1000 and centred on the boat position. thnx, Paddy

**Comment by antipole2** [comment]
v3 of the plugin has the ability to centre the chart where you will and set the scale, zooming in or out.
I am using this to drop a mark at a location on the clipboard.

Keyboard short-cuts are tricky as OpenCPN gets these and, in any event, there could be multiple scripts competing for the keystroke.  In v3 you can click on the console close button to get the script's attention.  I use this to say "Look on the clipboard, drop a mark at that location, centre the chart there and zoom in if need be so the mark is displayed".  Works well.

**Comment by antipole2** [comment]
@boatybits Are you good now?  Can we close this?

**Comment by antipole2** [comment]
Not heard further from @boatybits after 8 months, so closing.
